### ISSUE TITLE
Upgrade to windows 10

### ISSUE THREAD

**Issue by kayanow2** [Top-Cat/UnofficialSteamAuthenticator#74]
Hi, I upgraded to windows 10 from 8.1 without thinking about this app (Doh). It wont open now but I think it's meant to work with windows 10 it just says pending. Should I reinstall or do I need to  revoke first, can anyone advise please.

**Comment by kayanow2** [comment]
OK I managed to do this I know it's easy but just in case anyone else finds themselves in a similar position, I had steam on my PC logged in permanantly so make  sure to revoke the 8.1 authenticator via your PC connection so you don't need to log in, then I had to restart my phone because it didn't recognise that the app. had been uninstalled. Re install the app. and re set up not forgetting to make a note of your new revoke code. So easy really and I don't know why I was so worried. Good job I was logged in to steam on my PC otherwise it would have been more complicated.
